app/javascript/packages/document-capture-polling/index.ts

```diff
@@ -41,12 +43,18 @@ export class DocumentCapturePolling {
 
   statusEndpoint: string;
 
-  trackEvent: typeof defaultTrackEvent;
+  trackEvent: typeof defaultTrackEvent = (event, payload) =>
+    defaultTrackEvent(event, {
+      ...payload,
+      phone_question_ab_test_bucket: this.phoneQuestionAbTestBucket,
```

Comment:
I don't see where `this.phoneQuestionAbTestBucket` is actually being set, though I wouldn't be surprised if past @aduth added some obscure magic. I'd expect to see something in the `constructor` to receive and assign this property.

Reply:
I think it's more likely that I missed this since I'm more familiar with functional components. Added to the consructor ([commit](https://github.com/18F/identity-idp/pull/9400/commits/021693ef11008887a8037c8acd4780b4bd2e9a23)).

Reply:
>Edit: Also, I'm going to go look at the tests for this.

Thanks!